src/ui/components/chapterlabel/ChapterLabel.tsx

```diff
@@ -0,0 +1,56 @@
+import type { StyleProp, TextStyle } from 'react-native';
+import { Text } from 'react-native'
+import React, { useCallback, useContext, useState } from 'react';
+import { PlayerEventType, TimeUpdateEvent } from 'react-native-theoplayer';
+import { PlayerContext, UiContext } from '../util/PlayerContext';
+import { useChaptersTrack } from '../../hooks/useChaptersTrack';
+import { usePlayerEvent } from '../../hooks/usePlayerEvent';
+
+export interface ChapterLabelProps {
+  /**
+   * The style overrides.
+   */
+  style?: StyleProp<TextStyle>;
+}
+
+export interface ChapterLabelState {
+  currentTime: number;
+}
+
+/**
+ * The default style for the time label.
+ */
+export const DEFAULT_CHAPTER_LABEL_STYLE: TextStyle = {
+  marginLeft: 10,
+  marginRight: 10,
+  height: 20,
+  alignSelf: 'center',
+};
+
+/**
+ * The default time label component for the `react-native-theoplayer` UI.
+ */
+
+
+export const ChapterLabel = (props: ChapterLabelProps) => {
+    const player = useContext(PlayerContext).player;
+    const [currentTime, setCurrentTime] = useState(0);
```

Comment:
We already have a [`useCurrentTime`](https://github.com/THEOplayer/react-native-theoplayer-ui/blob/v0.11.0/src/ui/hooks/useCurrentTime.ts) hook for this.